ansible/roles/passenger/tasks/main.yml

```diff
@@ -22,19 +22,37 @@
   become: true
   when: passenger_repo.changed
 
-- name: install nginx and passenger packages
+- name: install nginx for 20.04
+  apt: pkg=nginx state=latest
+  become: true
+  when: ansible_distribution_version == "20.04"
+
+- name: install passenger module for 16.04
+  apt: pkg=passenger state=latest
+  become: true
+  when: ansible_distribution_version == "16.04"
+
+- name: install passenger module for 20.04
+  apt: pkg=libnginx-mod-http-passenger state=latest
+  become: true
+  when: ansible_distribution_version == "20.04"
```

Comment:
Can we group the 20 stuff together and the 16 stuff together, and add a couple comments of what's what, what to clean up in the future, etc, since we're in transition?